spec/apiForSheetsSpec.js

```diff
@@ -250,39 +250,81 @@ describe('Api For Sheets Module', () => {
+    ' pull request is at oppia repository',
+    async () => {
+      // Add username to CLA sheet.
+      await apiForSheetsModule.generateOutput(
+        [...claData, ['testuser7777']],
+        pullRequestPayload.payload.pull_request.number,
+        pullRequestPayload.payload.pull_request
+      );
+
+      var linkText = 'here';
+      var linkResult = linkText.link(
+        'https://github.com/oppia/oppia/wiki/Contributing-code-to-' +
           'Oppia#setting-things-up'
-        );
-        expect(github.issues.createComment).toHaveBeenCalledWith({
-          owner: pullRequestPayload.payload.repository.owner.login,
-          repo: pullRequestPayload.payload.repository.name,
-          number: pullRequestPayload.payload.pull_request.number,
-          body:
-            'Hi! @testuser7778, Welcome to Oppia! Please could you ' +
-            'follow the instructions ' + linkResult +
-            ' to get started? You\'ll need to do ' +
-            'this before we can accept your PR. ' +
-            'I am closing this PR for now. Feel free to re-open it ' +
-            'once you are done with the above instructions. Thanks!',
-        });
-        expect(github.issues.update).toHaveBeenCalledWith({
-          issue_number: pullRequestPayload.payload.pull_request.number,
-          owner: pullRequestPayload.payload.repository.owner.login,
-          repo: pullRequestPayload.payload.repository.name,
-          state: 'closed',
-        });
+      );
+
+      expect(github.issues.createComment).toHaveBeenCalledWith({
+        owner: pullRequestPayload.payload.repository.owner.login,
+        repo: pullRequestPayload.payload.repository.name,
+        number: pullRequestPayload.payload.pull_request.number,
+        body: 'Hi! ' + '@testuser7778,' +
+        ' Welcome to Oppia! Please could you follow the instructions ' +
+        linkResult + ' to get started ? You\'ll need to do this' +
+        ' before we can accept your PR. I am closing this PR  for' +
+        ' now. Feel free to re-open it once you are done the' +
+        ' above instructions. Thanks!',
       });
+      expect(github.issues.update).toHaveBeenCalledWith({
+        issue_number: pullRequestPayload.payload.pull_request.number,
+        owner: pullRequestPayload.payload.repository.owner.login,
+        repo: pullRequestPayload.payload.repository.name,
+        state: 'closed',
+      });
+    });
+
+    describe('when pull request created at oppia-android repository', () => {
+      beforeEach(function (done) {
+        pullRequestPayload.payload.repository.name = 'oppia-android';
+        done();
+      });
+
+      it('should close the PR if atleast one user has not signed cla',
+        async () => {
+        // Add username to CLA sheet.
+          await apiForSheetsModule.generateOutput(
+            [...claData, ['testuser7777']],
+            pullRequestPayload.payload.pull_request.number,
+            pullRequestPayload.payload.pull_request
+          );
```

Comment:
This function expects the repository name to be passed as well.

Reply:
addressed in the latest commit